src/main/java/org/folio/processor/RuleProcessor.java

```diff
@@ -65,6 +67,7 @@ public RuleProcessor(TranslationHolder translationHolder) {
    * @return content of the generated marc record
    */
   public String process(EntityReader reader, RecordWriter writer, ReferenceDataWrapper referenceData, List<Rule> rules, ErrorHandler errorHandler) {
+    log.info("process:: the rules param: {}",rules);
```

Comment:
Will rules be changed between this method and next log entry to make its logging twice?